mlu_op.h

```diff
@@ -13253,6 +13253,271 @@ mluOpDCNBackwardData(mluOpHandle_t handle,
                      const mluOpTensorDescriptor_t grad_mask_desc,
                      void *grad_mask);
 
+// Group:FFT
+/*! @brief The descriptor of FFT (Fast Fourier Transform) operation that holds FFT information including
+
+    the tensor descriptor of input tensor and output tensor, the rank of FFT, the FFT size on each
+    dimension, the size of reserved space and the size of workspace.
+    *
+    You need to call the ::mluOpCreateFFTPlan function to create a descriptor for the FFT operation, and call
+    the ::mluOpMakeFFTPlanMany function to set the information of the FFT operation to the descriptor.
+    Then, you need to allocate the reserved space and set the space to the FFT descriptor by ::mluOpSetFFTReserveArea.
+    AT the end you need to destroy the Cambricon MLUOP context with the ::mluOpDestroyFFTPlan.
+    */
+typedef struct mluOpFFTStruct *mluOpFFTPlan_t;
+
+// Group:FFT
+/*!
+
+    @brief Creates a descriptor pointed by \p fft_plan for the FFT operation, and allocates memory
+    for holding the information about the FFT operation. The information is defined in ::mluOpFFTPlan_t.
+    *
+    @param[out] fft_plan
+    Output. Pointer to the FFT descriptor that holds information about the FFT operation.
+    *
+    @par Return
+    - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_ALLOC_FAILED
+    *
+    @par API Dependency
+    - After calling this function, you can call the ::mluOpMakeFFTPlanMany function to initialize and set the
+    information to the created descriptor.
+    - You need to call the ::mluOpDestroyFFTPlan to destroy the descriptor.
+    Otherwise, the memory leak may occur.
+    *
+    @par Note
+    - This function only supports 1D FFT currently. 2D FFT and 3D FFT
+    will be supported in the future.
+    - When the data type of input is float or complex_float, the 1D FFT length should be equal to:
+    length = \f$base * 2^ {m}\f$ , and the base should be less than or equal to 4096.
+    - When the data type of input is half or complex_half, the 1D FFT length should be equal to:
+    length = \f$2^{m}
+
+    \f$.
+    *
+    @par Requirements
+    - None.
+    *
+    @par Example
+    - None.
+    */
```

Comment:
```suggestion
 * @brief Creates a descriptor pointed by \p fft_plan for the FFT operation, and allocates memory
 * for holding the information about the FFT operation. The information is defined in ::mluOpFFTPlan_t.
 *
 * @param[out] fft_plan
 * Output. Pointer to the FFT descriptor that holds information about the FFT operation.
 *
 * @par Return
 * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_ALLOC_FAILED
 *
 * @par API Dependency
 * - After calling this function, you can call the ::mluOpMakeFFTPlanMany function to initialize and set the
 * information to the created descriptor.
 * - You need to call the ::mluOpDestroyFFTPlan to destroy the descriptor.
 * Otherwise, the memory leak may occur.
 *
 * @par Note
 * - This function only supports 1D FFT currently. 2D FFT and 3D FFT
 * will be supported in the future.
 * - When the data type of input is float or complex_float, the 1D FFT length should be equal to:
 * length = \f$base * 2^ {m}\f$ , and the base should be less than or equal to 4096.
 * - When the data type of input is half or complex_half, the 1D FFT length should be equal to:
 * length = \f$2^{m}
 *
 * \f$.
 *
 * @par Requirements
 * - None.
 *
 * @par Example
 * - None.
 */
```
后面的接口注释请按照此格式修改